dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/OpenAireExportUtil.java

```diff
@@ -841,6 +843,12 @@ public static void writeDatesElement(XMLStreamWriter xmlw, DatasetVersionDTO dat
         writeEndTag(xmlw, date_check);
     }
 
+    private static <K, V> Map<K,V> createMap(K key, V value) {
+        Map<K, V> result = new HashMap<>();
+        result.put(key, value);
+        return result;
+    }
```

Comment:
The problem here is that the method where this map is passed treats it as a mutable map, and it can happen (not in my cases though), that it would try to put something into. That's why I've created the method.

Reply:
Ok. I didn't look close enough.